cfgov/v1/jinja2/v1/includes/molecules/call-to-action.html

```diff
@@ -38,8 +38,7 @@ <h2 class="a-heading">
     {% endif %}
 
     {% if value.button %}
-    <a class="a-btn a-btn__full-on-xs
-             {{ 'a-btn__super' if value.button.size == 'large' else '' }}"
+    <a class="a-btn a-btn__full-on-xs"
```

Comment:
The Hyperlink module includes an `aria-label` option; see [hyperlink.html](https://github.com/cfpb/consumerfinance.gov/blob/b6417fce0429c5b81dc0c803d431db6bbc300634/cfgov/v1/jinja2/v1/includes/atoms/hyperlink.html#L22). Can we use that here in case one is specified?

More generally I wonder if we can just use that template directly, via `{% include_block value.button %}` - I note that template includes `a-link__jump` by default which is a little surprising.